backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/CpuPinningPolicy.java

```diff
@@ -43,4 +43,38 @@ public int getValue() {
     public boolean isExclusive() {
         return this == DEDICATED || this == ISOLATE_THREADS;
     }
+
+    public static int compare(CpuPinningPolicy policy, CpuPinningPolicy other) {
```

Comment:
What about implementing a `Comparable` interface instead of adding a static method?

Reply:
> What about implementing a `Comparable` interface instead of adding a static method?

I wished to. But the class is ENUM and `Comparable` interface has a final method `compareTo` for ENUM class, which I can't override.

Reply:
Ok, thanks for the explanation. In that case, I do not see much benefit in putting it to `CpuPinningPolicy` and I'd prefer to move it back to `VmsCpuPinningPolicyComparator`. There is onlly a very little chance it will be used outside of that class.

Reply:
Ok, we can leave it here. It is not a bad place for the compare code, I'd just prefer to put it in the comparator.